integration/__tests__/app.tests.ts

```diff
@@ -505,19 +508,35 @@ describe("Lens integration tests", () => {
         await app.client.click(".sidebar-nav [data-test-id='workloads'] span.link-text");
         await app.client.waitUntilTextExists('a[href^="/pods"]', "Pods");
         await app.client.click('a[href^="/pods"]');
+        await app.client.click(".NamespaceSelect");
+        await app.client.keys("kube-system");
+        await app.client.keys("Enter");// "\uE007"
         await app.client.waitUntilTextExists("div.TableCell", "kube-apiserver");
+        let podMenuItemEnabled = false;
+
+        // Wait until extensions are enabled on renderer
+        while (!podMenuItemEnabled) {
```

Comment:
Fine for this PR but I think we need to have a better fix for this in future. Buttons should appear when an extension gets loaded.

Reply:
Agree